spec/support/analyzer/97_ignore_by_comment.rb

```diff
@@ -0,0 +1,16 @@
+def not_ignored_rescue_vs_respond
+  begin
+    'abakus'.to_a
+  rescue NoMethodError
+
+  end
+end
+
+# :rescue_vs_respond_to
+def ignored_rescue_vs_respondd
+  begin
+    'abakus'.to_a
+  rescue NoMethodError
+
+  end
+end
```

Comment:
what would happen if there was a third method that rescued the no method error as well but was not ignored?

Reply:
I guess the third one would be accounted for as there is the reset_comment method.